src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -61,6 +67,36 @@ static constexpr Property<std::string, PropertyMutability::WO> config_device_id{
 static constexpr Property<ov::threading::IStreamsExecutor::ThreadBindingType, PropertyMutability::RW> cpu_bind_thread{
     "CPU_BIND_THREAD"};
 
+/**
+ * @brief Number of streams in Performance-core(big core)
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> big_core_streams{"BIG_CORE_STREAMS"};
+
+/**
+ * @brief Number of streams in Efficient-core(small core) on hybrid cores machine
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> small_core_streams{"SMALL_CORE_STREAMS"};
+
+/**
+ * @brief Number of threads per stream in big cores
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> threads_per_stream_big{"THREADS_PER_STREAM_BIG"};
+
+/**
+ * @brief Number of threads per stream in small cores on hybrid cores machine
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> threads_per_stream_small{"THREADS_PER_STREAM_SMALL"};
+
+/**
+ * @brief Small core start offset when binding cpu cores
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> small_core_offset{"SMALL_CORE_OFFSET"};
```

Comment:
BTW, maybe we can move these properties to stream executor configuration? E.g. to this file https://github.com/openvinotoolkit/openvino/blob/c3ec58a09bbe33ffade8975809e5c76bed53c78e/src/inference/dev_api/openvino/runtime/threading/istreams_executor.hpp

Reply:
These properties are internal properties for `threading`, seems put them into threading directory does make sense, but put them into this internal_properties.hpp isn't harmful, and previous some threading related properties also has been put into this header file, such as `threads_per_stream{"THREADS_PER_STREAM"}` and `cpu_bind_thread{
    "CPU_BIND_THREAD"}`
@wangleis how do you think? Do we need move all `threading` properties into threading's header file?

Reply:
@ilya-lavrenov @riverlijunjie There are two kinds of threading related properties in CPU plugin. One is CPU plugin properties for application. The other is StreamsExecutor properties which may be used by OV core and plugins. 

MT 2.0 has enabled stream info table to replace many StreamsExecutor properties for MT 1.0. And we will clean legacy MT 1.0 code included those StreamsExecutor properties in following PRs. So moving all StreamsExecutor properties to StreamsExecutor header file or leaving them to be cleaned by following are both ok for me.

Reply:
Move threading's properties into `threading/istreams_executor.hpp`. @ilya-lavrenov @wangleis